internal/pkg/overlay/overlay.go

```diff
@@ -272,6 +273,110 @@ func (overlay Overlay) Mkdir(path string, mode int32) (err error) {
 	return os.MkdirAll(fullPath, os.FileMode(mode))
 }
 
+// returns a list of the variable names for the given template
+func (overlay Overlay) ParseVars(file string) []string {
+	if !strings.HasSuffix(file, ".ww") {
+		return nil
+	}
+	fullPath := overlay.File(file)
+	if !util.IsFile(fullPath) {
+		wwlog.Error("Template file does not exist in overlay %s: %s", overlay.Name(), file)
+		return nil
+	}
+
+	var (
+		writeFile  bool
+		backupFile bool
+	)
```

Comment:
Since these are never used, they can be removed if getTemplateFuncMap call signature is updated as suggested.